lib/msal-react/docs/hooks.md

```diff
@@ -145,7 +145,7 @@ The `useMsalAuthentication` hook will initiate a login if a user is not already
 ### Input Parameters
 
 - [interactionType](https://azuread.github.io/microsoft-authentication-library-for-js/ref/enums/_azure_msal_browser.interactiontype.html) (Popup, Redirect, or Silent) specifies how you would like to acquire tokens or login when interaction is required (note the Silent option has some extra considerations explained below)
-- [request object](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-react-feature-branch/lib/msal-browser/docs/request-response-object.md#request) (optional) specifies additional parameters to be used by the login or token acquisition call
+- [request object](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/request-response-object.md) (optional) specifies additional parameters to be used by the login or token acquisition call
```

Comment:
The request-response-object.md is a good reference point because it has a table that includes all the request objects with a link to each request in the typedoc.